spring-cloud-gcp-samples/spring-cloud-gcp-integration-pubsub-sample/README.adoc

```diff
@@ -1,13 +1,26 @@
 = Spring Integration Channel Adapters for Google Cloud Pub/Sub Code Sample
 
 This code sample contains a couple of apps.
-One app gets a message provided by the user through a webpage and publishes it to a Google Cloud Pub/Sub topic.
-The other app subscribes to that topic and logs received messages.
 
+* Pub/Sub Sender & Receiver Applications:
++
+The `sample-sender` and `sample-receiver` applications demonstrate how to send messages from one
+application to another using Pub/Sub. The `sample-sender` provides a web interface to send messages
+to a topic whose messages are received by the `sample-receiver`.
```

Comment:
I don't know how to make it automatic an reject during project build, but our preference is to use `One sentence per line` rule: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line